packages/sources/src/components/card/card.models.ts

```diff
@@ -0,0 +1,17 @@
+import { Button } from "../button/button.models";
+
+export interface Card<TemplateFnReturnType> {
+  label: string;
+  content: TemplateFnReturnType | string;
```

Comment:
Opzich kan ik wel leven met deze suggestie, maar laten we 'm dan ook in deze twee varianten waarborgen. Alternatief is geen union, en dan is mijn voorkeur voor `TemplateFnReturnType`. Hoe denk jij hierover?

Reply:
Vanwege een change hierboven, is deze `TemplateFnReturnType` niet meer nodig. Maar laten we hier samen nog eens naar kijken of dat _echt_ wel zo is